website/src/pages/docs/server-side-rendering.mdx

```diff
@@ -116,6 +116,18 @@ The `extractor.getScriptTags()` returns a string of multiple `<script>` tags mar
 
 Alternatively the `ChunkExtractor` also has a `getScriptElements()` method that returns an array of React elements.
 
+### Entrypoints
+
+By default, `ChunkExtractor` will collect chunks from the `main` entrypoint (which is the default behaviour of `webpack`), if you would
```

Comment:
This is not exact, we use `entrypoints` as `assets, we are not collecting chunks from them.

Please remove this section.

Reply:
Well, what [getMainAssets](https://github.com/smooth-code/loadable-components/blob/master/packages/server/src/ChunkExtractor.js#L331) is doing then? Main entry point might depend on some extra chunks, and that have to be counted separately from "loadable"-ed chunks.
As long as these "entry" chunks and chunks used by `NamedAssets` could overlap - this sentence is 100% true.

Reply:
Okay, let me dig into the problem once more, because with my webpack configuration, loadable didn't work till I specified `entrypoints`. I will get back with feedback, in the meantime, we can discuss.

Thanks for all your feedback.

Reply:
My webpack config entries for client/server are specified this way:

Client
```
entry: {
   client: './src/client.tsx',
},
```

Server
```
entry: {
   server: './src/server.tsx',
},
```

If they were this way:

Client
```
entry: './src/client.tsx'
```

Server
```
entry: './src/server.tsx'
```

It will work because webpack will call those entries `main.js`, but otherwise it errors saying:

```
Invariant Violation: loadable: cannot find main in stats
```

I think this case should be specified in the docs, as it took me several days to find out the problem was coming from `loadable-stats.json` (for some reason I thought `loadable` was getting entries from `webpack` dynamically).

Let me know if you agree on this. 
I will also rephrase this section of the docs being more specific on this case.

Reply:
Many people spend quite a lot of time here. I've asked :)

Reply:
@neoziro @theKashey I've rephrased `Entrypoints` in what I think is a better explanation of the problem.

Let me know what you think and if I need to change something.

Thanks!